recipes/boost-leaf/all/conanfile.py

```diff
@@ -0,0 +1,91 @@
+from conan import ConanFile
+from conan.tools.files import get, copy
+from conan.tools.layout import basic_layout
+from conan.tools.build import check_min_cppstd
+from conan.errors import ConanInvalidConfiguration
+import os
+
+
+required_conan_version = ">=1.50.0"
+
+
+class BoostLEAFConan(ConanFile):
+    name = "boost-leaf"
+    version = "1.81.0-prerelease"
```

Comment:
```suggestion
```
Version should not be part of the recipe, so we can re-use the same recipe for any version.

Reply:
Understood. I'll remove that line. So this is a situation where we can't use 1.80.0 because of how Boost does versioning. Mainly that Boost has release cycles and any Boost library with an update puts their code forward for that version. Since there isn't a Boost.Leaf version for 1.80 it wouldn't in sync with Boost versions. And version 1.81 hasn't been cut yet. But I understand that this is not ideal. I'll close this PR for now and make the fixes when version 1.81 is cut.